### ISSUE TITLE
I can confirm that the issue about application crashes is still an issue for 14.4 (iphone 7).  In particular, attaching to imagent process results in a crash

### ISSUE THREAD

**Issue by kye4u2** [frida/frida#1671]
I can confirm that the issue about application crashes is still an issue for 14.4 (iphone 7).  In particular, attaching to imagent process results in a crash

_Originally posted by @kye4u2 in https://github.com/frida/frida/issues/1601#issuecomment-802919665_

**Comment by gprime31** [comment]
Same on Andriod 8.1

**Comment by wrlu** [comment]
Yes I have meet a same issue on my iPhone 7 Plus with iOS 14.3, App freezes and crashes when hooking.

_Originally posted by @kye4u2 in https://github.com/frida/frida/issues/1601#issuecomment-802919665_
